### ISSUE TITLE
Work items created in Analytics Insights can't be opened in Chromium and Firefox.

### ISSUE THREAD

**Issue by pkettman** [openshiftio/openshift.io#3226]
The work item created in the Analytics Insights menu can't be opened, see blank screen and Chrome log: "Redirect ... was blocked by CORS policy...". And I can't see it in the Planner backlog.

Scenario:
1) Open 'View complete stack report' from 'Stack Report Recommendations'
2) Select 'Insights' and tab 'Companion Dependency Details'
3) Select a dependency and click 'Create work item'
4) Click 'View here' after the WI has been created 
5) I got forwarded to a black page, see picture below
6) In addition the WI is not present in the Planner backlog

Tested on Chrome 56.0.2924.87 Fedora Project (64-bit) and Firefox.

![insights_wi_created](https://user-images.githubusercontent.com/26871874/39049800-3bdc15d8-44a3-11e8-8164-bf283491e1ec.png)
![insights_wi_view](https://user-images.githubusercontent.com/26871874/39049824-4f944cd0-44a3-11e8-8a5a-892417a028f5.png)

**Comment by ldimaggi** [comment]
Peter - can you list the precise set of actions that you performed to encounter this bug? I just tried (with a REST/VertX quickstart) and was not able to recreate the bug:

![screenshot from 2018-04-20 10-55-10](https://user-images.githubusercontent.com/642621/39058413-bf31f944-4489-11e8-8e42-c4d3e7ae51d0.png)

**Comment by pkettman** [comment]
Len, I added the scenario above and recreated it also on Firefox.

**Comment by ldimaggi** [comment]
Hmm - the only difference is that I was able to access the workitem through the dependencies panel - does that work for you?

Which quickstart did you use?

**Comment by ldimaggi** [comment]
Hey Peter - I am still able to create and access a workitem - this  time from the "insights" section - can you tell me exactly which quickstart mission and runtime you used to create the issue? thx!

**Comment by miteshvp** [comment]
@pkettman - I am not able to reproduce this issue. It works as expected for me. Can you please try a new stack?

**Comment by pkettman** [comment]
@ldimaggi I used Eclipse Vert.x Booster, created in February, see https://github.com/pkettman/pke-demo21/ for details. Let's spend some time on this one on our today's session.

**Comment by pkettman** [comment]
@miteshvp I've tried another codebase of the same stack (Vert.x, https://github.com/pkettman/pke-demo22/) - got the same result. Another stack (newly created WildFly Swarm) doesn't offer any dependencies to create a WI for.
Interestingly when I go to the a stack analysis where I've previously created a WI, I am allowed to create the WI again, see below. So I will test it with Len today.
![insights_wi2](https://user-images.githubusercontent.com/26871874/39124220-f10a24aa-46fa-11e8-920e-7cf9e705df0b.png)

**Comment by pkettman** [comment]
It work for me in Chrome, but not in Chromium and Firefox.

**Comment by pkettman** [comment]
@invincibleJai I retested the scenario today and it's still not working as expected, see below:
1) I created two new work items over 'Create work item' in the 'Companion Dependency Details' tab. Links are already working fine.
![wi-created](https://user-images.githubusercontent.com/26871874/44976793-92401b80-af66-11e8-98b5-62b9a422bdcf.png)

2) went to planner to see them, but it can't be opened it at all (waiting 3-5 minutes)
![wi-unaccessible in planner](https://user-images.githubusercontent.com/26871874/44976801-9704cf80-af66-11e8-9b42-8e0c05e7a155.png)

3) went back to 'Companion Dependency Details' tab for the same project and I am again allowed to create WIs for the same dependencies. The previously created ones a not linked there as before. 
![wis not visible on analytics](https://user-images.githubusercontent.com/26871874/44976806-99ffc000-af66-11e8-88b7-4369b99be27c.png)

**Comment by animuk** [comment]
@pkettman I have just tested on my chrome version 68.0.3440.106
![image](https://user-images.githubusercontent.com/11207106/45091514-53f65800-b130-11e8-869c-8738c3a350e1.png)
and for me the scenario is working perfectly fine. Please see the screenshot below:

![image](https://user-images.githubusercontent.com/11207106/45091568-830cc980-b130-11e8-9d40-d09ab21ba0a3.png)
![image](https://user-images.githubusercontent.com/11207106/45091582-9029b880-b130-11e8-815f-638edfca9239.png)

Might be looks like you have some browser issue, try reinstalling the latest version of the chrome

**Comment by invincibleJai** [comment]
Let my try in fedora as well once.

**Comment by pkettman** [comment]
I've just reinstalled Chrome to Version 69.0.3497.81, but the described issue is still there. After an initial creation are the WIs shown in the 'Companion Dependency Details' tab by each dependency, but when I leave this screen (e.g. go to pipelines, etc.) and come back, they are not there, see: 
![wis not visible on analytics2](https://user-images.githubusercontent.com/26871874/45093958-4d5ce600-b11a-11e8-8475-05a969991f4c.png)

**Comment by animuk** [comment]
@pkettman I think, this functionality is not implemented. @invincibleJai can you check

**Comment by invincibleJai** [comment]
@pkettman there are two things we are discussing here i believe

1. Work items created in Analytics Insights can't be opened in Chromium and Firefox (as title of issue says) 
Expectation: User clicks on "Create work item" under the companion tab, it should show toast notification with the hyperlink "View work item". If user clicks on view work item, it should open a new window in browser and user can see Work item in the planner.

Actual: I tried this on chrome/firefox/chromium om mac and it's working as expected.

Does this scenario work fine for you?

** Note : i'll try in fedora as well and update thread.

2. Once the user creates a workitem via stack report and goes to other flow i.e leaves stack report screen and revisit it, already created work items are not shown.

This we are aware currently we(Analytics) don't persist the created work item status so once user leaves the screen and revisit already created work items are not there. 

Let us know if first one is working for you.

CC @miteshvp

**Comment by ravsa** [comment]
@pkettman @invincibleJai I tried on chromium, Firefox, Chrome and things are working fine for me.

```bash
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	Fedora
Description:	Fedora release 28 (Twenty Eight)
Release:	28
Codename:	TwentyEight
```
![screenshot from 2018-09-06 12-33-33](https://user-images.githubusercontent.com/11839674/45141021-57dab680-b1d2-11e8-9d53-59de76ed6be6.png)
![screenshot from 2018-09-06 12-31-37](https://user-images.githubusercontent.com/11839674/45141027-5ad5a700-b1d2-11e8-98c1-f99ae5c90af0.png)
![screenshot from 2018-09-06 12-25-47](https://user-images.githubusercontent.com/11839674/45141035-6032f180-b1d2-11e8-9fee-d6a0f231452e.png)

**Comment by pkettman** [comment]
@invincibleJai yes, there are two things discussed here:
1) the original scenario works already fine for me on both Chrome and Firefox
2) this is still an open issue - the 3 steps scenario is in my comment from Sep 3rd

Are you fine with resolving item 2) here or do you prefer to close this ticket and open another one for 2)?

**Comment by miteshvp** [comment]
@pkettman - item 2 described above is more of a feature request. Hence I would request you to close this issue and create a new issue with feature request for team-analytics. cc @invincibleJai

**Comment by pkettman** [comment]
@miteshvp Closing this ticket as the scenario 1 works fine. I am going to open a separate bug for the scenario 2. I am not sure what you meant by a feature request, but when a user goes repeatedly to 'Companion Dependency Details' tab and can't see links to WIs he created and were there on his/her previous page visit, it's definitely a wrong behavior - bug.
